src/main/java/ui/CreateRecipeUI.java

```diff
@@ -79,15 +85,13 @@ public void actionPerformed(ActionEvent e) {
                     JOptionPane.showMessageDialog(null,
                             "Pleas enter an INTEGER for calories, time, difficulty!");
                 }
-                RecipeInputBoundary recipeInputBoundary = new RecipeInterActor(recipeOutputBoundary);
-                RecipeController recipeController = new RecipeController(recipeInputBoundary);
 
                 String ingredient = ingredients.getText();
                 String[] ingredients_list = ingredient.split(",");
                 ArrayList<String> ingredients = new ArrayList<>(Arrays.asList(ingredients_list));
```

Comment:
Same as the comment above, ingredients_list follows Python's naming convention, I think ingredientsList would be a better option.